send/ADConnector.pm

```diff
@@ -118,6 +127,59 @@ sub init_config($) {
 
 }
 
+#
+# Resolve available domain controlers from a URL
+#
+sub resolve_domain_controlers($) {
+	my $ldap_location = shift;
+	my ($protocol, $host, $port);
+	if($ldap_location =~ /^([^\/]+:\/\/)?([^:]+):([0-9]+)$/) {
```

Comment:
Usually you are using a space between "if" and "(". It's good to be consistent.

Reply:
Fixed